containerd/containerd.go

```diff
@@ -67,6 +67,15 @@ var ContainerdCommand = cli.Command{
 			containerdDir = stateDir
 		}
 
+		defaultCpu := context.GlobalInt("default_cpus")
+		if defaultCpu <= 0 {
+			defaultCpu = 1
+		}
+		defaultMemory := context.GlobalInt("default_memory")
+		if defaultMemory <= 0 {
+			defaultMemory = 128
+		}
+
```

Comment:
could you remove this block of code and move the "if" blocks into supervisor.New()?